packages/back-end/src/util/mssqlPoolManager.ts

```diff
@@ -0,0 +1,14 @@
+import mssql from "mssql";
+import { MssqlConnectionParams } from "../../types/integrations/mssql";
+const pools = new Map();
+
+export function get(name: string, config: MssqlConnectionParams) {
```

Comment:
Naming suggestion: Since this also can create a connection ... maybe something like `findOrCreateConn` ?